### PR TITLE
Scrum s303 save export results

### PR DESCRIPTION
**Pull request by mhyneman8**
### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description

We created a download results button. This will allow the user to take their calculator results to someone else or print for future use.

html2canvas 
jspdf

These two work together to be able to download the component we made. jspdf creates the pdf and html2canvas keeps the styling like the page.

There is no ticket, because it was deleted during the sprint

### Screenshots

Include relevant screenshots.

#### Before
image

#### After
image

localhost_3000_calculator_offense-fish-5-eligible

### Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings in the console
- [x] There are no new linting errors/problems in my code
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: components/helper/Results.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
+import {
+  Box, Button,
+  Container,
+  Typography,
+} from '@mui/material';
+import Image from 'next/image';
+// import React, { useState } from 'react';
```
Comment (e-papanicolas) [inline components/helper/Results.tsx:7]: Remove extra commented out import.

File: components/helper/Results.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+import React from 'react';
+
+import Theme from '../../styles/themes/theme.tsx';
+// import {ImageContainer} from '../layout/ImageContainer.jsx';
+import SectionContainer from '../layout/SectionContainer.tsx';
+
+type Answers = {
+    [key: string]: string;
+}
```
Comment (e-papanicolas) [inline components/helper/Results.tsx:16]: Needs one extra line of spacer below.

File: components/helper/Results.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+  Box, Button,
+  Container,
+  Typography,
+} from '@mui/material';
+import Image from 'next/image';
+// import React, { useState } from 'react';
+import React from 'react';
+
+import Theme from '../../styles/themes/theme.tsx';
+// import {ImageContainer} from '../layout/ImageContainer.jsx';
```
Comment (e-papanicolas) [inline components/helper/Results.tsx:11]: The image container should be available now!

**Review by e-papanicolas — CHANGES_REQUESTED** [review]

**Comment by mhyneman8** [comment]
@e-papanicolas fixed the comments. The download is a little tough because it prints whatever is on the screen, which means it has to be a certain width to print appropriately. So I added an automatic close for screen sizes under 901. The pdf is on the screen long enough for it to be saved and then closes. 

Open to other ideas if you can think of them.

**Review by e-papanicolas — APPROVED** [review]
🥳

**Pull request merged**